lua/nvim-lsp-installer/servers/init.lua

```diff
@@ -14,11 +14,12 @@ local INSTALL_DIRS = {
     ["dockerls"] = "dockerfile",
     ["elixirls"] = "elixir",
     ["elmls"] = "elm",
-    ["eslint"] = "vscode-eslint",
     ["eslintls"] = "eslint",
+    ["eslint"] = "vscode-eslint",
     ["gopls"] = "go",
     ["hls"] = "haskell",
     ["intelephense"] = "php",
+    ["jsonnet"] = "go",
     ["kotlin_language_server"] = "kotlin",
```

Comment:
You can revert these diffs. No need to provide an entry for jsonnet here (see the comment above)

Reply:
I've backed out the change to this file.